scripts/seed-database/container.sh

```diff
@@ -14,7 +14,7 @@ REMOTE_PG="$2"
 psql --command="\\COPY (SELECT * FROM users) TO '/tmp/users.csv' (FORMAT CSV, DELIMITER ';')" "${REMOTE_PG}"
 
 # fetch teams
-psql --command="\\COPY (SELECT id, name, slug, theme, settings, domain, notify_personalisation FROM teams) TO '/tmp/teams.csv' (FORMAT CSV, DELIMITER ';')" "${REMOTE_PG}"
+psql --command="\\COPY (SELECT id, name, slug, theme, settings, notify_personalisation FROM teams) TO '/tmp/teams.csv' (FORMAT CSV, DELIMITER ';')" "${REMOTE_PG}"
```

Comment:
Having `domain` populated outside of prod means that email links don't point at the correct environment.